src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/tests/registries/RouteRegistry.test.js

```diff
@@ -10,11 +10,11 @@ test('Clear routes from RouteRegistry', () => {
     routeRegistry.addCollection([
         {
             name: 'route',
-            view: 'view',
             path: '/route',
             options: {},
             attributeDefaults: {},
             rerenderAttributes: [],
+            type: 'view',
```

Comment:
Why didn't we keep `view`? What we are describing here is always a `view`, and it would not be a BC break that way.

Reply:
This is incorrect the `/admin/config` request does not return `view` anymore since we refractor it and the `ViewRender` does render the [.type](https://github.com/sulu/sulu/blob/85f3a8fa62440e14d277724e20ca8b2e362758c1/src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/ViewRenderer.js#L32)  the `.view` is never accessed its false in the RouteConfig type decleration.

Reply:
@danrot the flow RouteConfig type (which is only used in the routeRegistry) was forgotten to be refractored when we changed this in `/admin/config` api that `routes` return `type` instead of `view`.